### PR TITLE
update archive node info into klaytn-docs

### PR DESCRIPTION
**Pull request by Sotatek-QuangDo**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by Sotatek-QuangDo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by ayo-klaytn — APPROVED** [review]

**Review by toniya-klaytn — APPROVED** [review]

File: docs/dapp/json-rpc/public-en.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
 |---|---|---|---|
 |[Klaytn API Service](https://www.klaytnapi.com/)| `https://public-node-api.klaytnapi.com/v1/cypress`|klay,eth,net|Full|
 |[Klaytn Foundation](https://www.klaytn.foundation)|`https://public-en-cypress.klaytn.net`|klay,eth,net|Full|
-|Fantrie|`https://cypress.fautor.app/archive`|klay,eth,net|Archive|
+|Fantrie|`https://archive-en.cypress.klaytn.net`|klay,eth,net|Archive|
```
Comment (markyim-klaytn) [inline docs/dapp/json-rpc/public-en.md:30]: This is provided by Klaytn Foundation not Fantrie

Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:30]:
```suggestion
|Klaytn Foundation|`https://archive-en.cypress.klaytn.net`|klay,eth,net|Archive|
```

File: docs/dapp/json-rpc/public-en.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
 |---|---|---|---|
 |[Klaytn API Service](https://www.klaytnapi.com/)| `wss://public-node-api.klaytnapi.com/v1/cypress/ws`|klay,eth,net|Full|
 |[Klaytn Foundation](https://www.klaytn.foundation)|`wss://public-en-cypress.klaytn.net/ws`|klay,eth,net|Full|
-|Fantrie|`wss://cypress.fautor.app/archive/ws`|klay,eth,net|Archive|
+|Fantrie|`wss://archive-en.cypress.klaytn.net/ws`|klay,eth,net|Archive|
```
Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:40]:
```suggestion
|Klaytn Foundation|`wss://archive-en.cypress.klaytn.net/ws`|klay,eth,net|Archive|
```

**Review by yeri-lee — APPROVED** [review]

File: docs/dapp/json-rpc/public-en.md (hunk 1 above)
Comment (Sotatek-QuangDo) [reply docs/dapp/json-rpc/public-en.md]: Yes I changed and updated the commit

File: docs/dapp/json-rpc/public-en.md (hunk 2 above)
Comment (Sotatek-QuangDo) [reply docs/dapp/json-rpc/public-en.md]: Yes I changed and updated the commit

**Review by toniya-klaytn — APPROVED** [review]

**Pull request merged**